tensorflow/contrib/lite/nnapi_delegate.cc

```diff
@@ -268,6 +268,23 @@ void AddOpsAndParams(tflite::Interpreter* interpreter,
       add_scalar_int32(builtin->activation);
     };
 
+    auto add_resize_bilinear_params = [&add_scalar_int32, &interpreter, &augmented_inputs](void* data) {
+      auto builtin = reinterpret_cast<TfLiteResizeBilinearParams*>(data);
+      if  (builtin->align_corners) {
```

Comment:
Is this only a check? if so mank builtin const, otherwise if it modifies it, its fine. but a small comment here about the side effects will be helpful.

Reply:
@shahzadlone Thanks for the suggestion. PR is rebased and updated as suggested.